modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -70,21 +73,18 @@ NAME                            STATUS
 ∙ image-digest-exporter-xlkn7   Completed
 ----
 
-. Retrieve the signature and payload from the object stored as `base64` encoded annotations:
+. Retrieve the signature from the object stored as `base64` encoded annotations:
 +
 [source,terminal]
 ----
 $ export TASKRUN_UID=$(tkn tr describe --last -o  jsonpath='{.metadata.uid}')
-$ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" > signature
-$ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/payload-taskrun-$TASKRUN_UID}" | base64 -d > payload
+$ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" | base64 -d > sig
```

Comment:
Per the guidelines, do not have more than one command per code block: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Please split these into two code blocks
![image](https://github.com/openshift/openshift-docs/assets/40172997/c9580242-13f2-40cb-8c69-c57249c49d2d)